tools/topology/sof-smart-amplifier.m4

```diff
@@ -222,7 +222,7 @@ DAI_CONFIG(ALH, eval(SMART_ALH_INDEX + 1), eval(SMART_BE_ID + 1), SMART_ALH_CAPT
 #SSP SSP_INDEX (ID: SMART_BE_ID)
 DAI_CONFIG(SSP, SMART_SSP_INDEX, SMART_BE_ID, SMART_SSP_NAME,
 	SSP_CONFIG(DSP_B, SSP_CLOCK(mclk, SSP_MCLK, codec_mclk_in),
-		      SSP_CLOCK(bclk, 9600000, codec_slave),
+		      SSP_CLOCK(bclk, 12288000, codec_slave),
 		      SSP_CLOCK(fsync, 48000, codec_slave),
 		      SSP_TDM(8, 25, 15, 255),
```

Comment:
if we change bitclk to 12.288M, the container size should be changed to 32 correspondingly, @RDharageswari can you help to confirm with @ryans-lee what format should we use?

Reply:
@keyonjie 
One question: if we have 12.288M clock, then we have 12.288M/48k = 256 available bits in frame. If we will use 25 container, we will have: 25 * 8 = 200 bits per frame, so SSP could be configured with 256-200=56 frame end padding. Are you sure that we should change also container size? Why we cannot use configuration with frame end padding?

Reply:
@keyonjie
I've updated this PR and changed bclk to 12288000 and container to 32 bits. Now, I think, configuration meets codec requirements: bclk/fclk = 256, container is 32 bit.